integration/angular_cli/package.json

```diff
@@ -22,7 +22,7 @@
     "@angular/router": "~9.1.1",
     "rxjs": "~6.5.4",
     "tslib": "^1.10.0",
-    "zone.js": "~0.10.2"
```

Comment:
This test project is still using Angular v9 which I believe doesn't support zone.js 0.11?

Reply:
I believe zone.js 0.11 still supports Angular v9, there should no breaking changes impact Angular v9.